styles/styles.css

```diff
@@ -404,7 +404,130 @@ main .section.highlight p {
   margin-bottom: 0;
 }
 
-/* Blog post styles */
+main .section.container, main .columns.container {
```

Comment:
I think here we would probably be better with a theme. So you could use `main .theme .container` instead of this complicated selector.